### DISCUSSION TITLE
v1.6.1

### DISCUSSION THREAD

**Discussion by lazy-immortal** [lazy-immortal/Sesame#11] [Announcements]
**Full Changelog**: https://github.com/lazy-immortal/Sesame/compare/v1.6.0.7...v1.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/lazy-immortal/Sesame/releases/tag/v1.6.1'>v1.6.1</a>.</em>

**Comment by jiaoye89** [comment]
建议优化通知常驻执行提醒的开关功能。
